app/mockServices/mockData/openshift-images.ts

```diff
@@ -21,7 +21,7 @@ export const imagesData = {
                     'description': 'Provides a Jenkins 1.X server on CentOS 7. For more information about using this container image, including OpenShift considerations, see https://github.com/openshift/jenkins/blob/master/README.md.',
                     'iconClass': 'icon-jenkins',
                     'openshift.io/display-name': 'Jenkins 1.X',
-                    'tags': 'jenkins',
+                    'tags': 'builder,jenkins',
```

Comment:
I added 'builder' to Jenkins so it would show up as the only item/sub-category, so I could test the not showing 'All' sub-category when there is only one sub-category.